src/client/chunk_closure.cpp

```diff
@@ -238,6 +239,14 @@ void ClientClosure::Run() {
             OnEpochTooOld();
             break;
 
+        case CHUNK_OP_STATUS::CHUNK_OP_STATUS_GET_AUTH_TOKEN_FAIL:
```

Comment:
If the token cannot be get, it is best to retry, such as when the third copy of mds is not online. IO hang instead of IO error will be better.  In addition, what if it just times out when getting the token?